src/pages/faq.astro

```diff
@@ -13,6 +13,16 @@ const content = [
     question: 'Something looks wrong - How can I debug?',
     answer: `You can debug Gitify by pressing <code>alt + command + I</code>. This will open the dev tools and then you can see any logs, network requests etc.`,
   },
+  {
+    id: '3',
+    question: 'How can I contribute to Gitify?',
+    answer: `You can contribute to Gitify by opening a pull request @ [gitify-app/gitify](https://github.com/gitify-app/gitify)`,
+  },
+  {
+    id: '4',
+    question: 'macOS is prompting for microphone access - what is this for?',
+    answer: `This is part of the permissions/entitlements when using the <code>Play sound</code> system setting.`,
```

Comment:
There's no way to separate it? Linking to something about that could help assuage concerns.

Reply:
If that's the place that says "can't ask for sound permission without also asking for mic permission," then yeah. Could also be a macos/plist doc? Not sure where the limitation is.

(This could also be a convenient reference for ourselves, if the underlying restriction changes, I suppose.)

Reply:
Makes sense.  If you happen to find a good web resource, let's incorporate it.  I had a quick search yesterday but couldn't find anything obvious

Reply:
Uh wait, then what is your answer based on? How do we know that's true?

Reply:
the dialog in https://github.com/gitify-app/gitify/issues/743

Reply:
My understanding of that dialog is different: there's a set of conditions that trigger the access request. It doesn't seem to indicate that the two are inseparable, which is what (to my understanding) this answer is saying.

Maybe I'm missing something, but I worry that this answer is indicating there's nothing we can do about it because of the way the permissioning system works, but we don't have evidence that's actually true.

Reply:
Removed from FAQ and believe I've found a way to remove from macOS plist file - https://github.com/gitify-app/gitify/pull/970

Reply:
Awesome! Glad I asked about it :). Great work tracking that down.